src/extension.ts

```diff
@@ -508,6 +517,34 @@ class Device extends vscode.TreeItem {
             [this.shellPort.toString()]);
         term.show();
     }
+    
+    async captureScreenshot() {
+        const error = (message) => {
+            vscode.window.showErrorMessage("Error capturing screenshot: " + message);
+        }
+        
+        try {
+            const screenshotDirectory = await getSharedTempDir('ev3dev-screenshots');
+            const screenshotFile = `${screenshotDirectory}/ev3dev-${sanitizedDateString()}.png`;
+
+            const conn = await this.exec('fbgrab -');
+            const writeStream = fs.createWriteStream(screenshotFile);
+            writeStream.on('open', () => {
+                conn.stdout.pipe(writeStream);
+            });
+
+            writeStream.on('close', () => {
+                vscode.window.setStatusBarMessage(`Screenshot successfully captured`);

```

Comment:
We should be, yes. And if either we make a mistake or VSCode has an issue, leaving the user without any sort of feedback as to what went on is a Bad Experience (TM).